File: Java/FuncionarioAssalariado.java
public class FuncionarioAssalariado extends Funcionario{



    public FuncionarioAssalariado(String nome,double salario){
        super(nome,salario);

    }

    @Override
    public void calcularSalario() {
        double salarioNormal = getSalario();
        System.out.println("Salário do funcionário " + getNome() + " R$: " + salarioNormal);
    }

}

File: Python/FuncionarioAssalariado.py
from Funcionario import Funcionario


class FuncionarioAssalariado(Funcionario):
    def __init__(self, nome, salario):
        super().__init__(nome, salario)

    def calcular_salario(self):
        salario_normal = self.get_salario()
        print(f"Salário do funcionário {self.get_nome()} é R$: {salario_normal}")

# Exemplos de us